app/cdash/xml_handlers/note_handler.php

```diff
@@ -115,7 +115,7 @@ public function endElement($parser, $name)
             } elseif (!$this->NoteCreator->name) {
                 \Log::error("Note missing name for build #{$this->Build->Id}");
             } elseif (!$this->NoteCreator->text) {
-                \Log::error("No note text for '{$this->NoteCreator->name}' on build #{$this->Build->Id}");
+                \Log::notice("No note text for '{$this->NoteCreator->name}' on build #{$this->Build->Id}");
```

Comment:
Would it make sense to use `Log::info()` here instead?  We currently don't use `Log::notice()` anywhere else in the codebase, and generally use `Log::info()` for these types of generic non-actionable messages.

As per [RFC 5424, Section 6.2.1, Table 2](https://datatracker.ietf.org/doc/html/rfc5424#section-6.2.1), the RFC Laravel's logging library follows, I believe this type of message would be classified as an informational message because it does not demand that the user take any actions.